pendulum_control/README.md

```diff
@@ -13,18 +13,18 @@ Run:
 
 ```
 . install/setup.bash
-pendulum_demo[__rmw_opensplice/__rmw_connext]
+pendulum_demo[__rmw_opensplice/__rmw_connextdds]
```

Comment:
opensplice is not supported anymore, so you can just delete it:

```suggestion
pendulum_demo[__rmw_connextdds]
```

Reply:
I applied these changes in [bd1935c](https://github.com/ros2/demos/pull/489/commits/bd1935ce0cb0706b033736afba022852fc91a18f), but I removed the square brackets since there is only one option for now, and I thought that would avoid possible confusion for users.

Reply:
ah yes, I didn't realize that removing the square brackets was better, thanks!